src/main/webapp/jsp/adminSearch.jsp

```diff
@@ -21,3 +22,13 @@
     
     <t:statusMessage/>
 </ta:adminPage>
+<!-- Formatting searchKey for highlight searchKey plugin ( ['string1','string2',...] )-->
+<c:set var="splittedSearchKey" value="${fn:split(data.searchKey, ' ')}" />		<!-- splitting by space	 -->
+<c:set var="formattedSearchKey" value='${fn:join(splittedSearchKey, \'","\')}' />				<!-- Joining with comma within single quotes -->
+<c:set var='formattedSearchKey' value='["${formattedSearchKey}"]' />									<!-- Surrounding string with single quote + square bracket -->
+
+<script type="text/javascript">
+ $( document ).ready(function() {
+	 $("body").highlight(${formattedSearchKey});
```

Comment:
@tarun14110 Could you put this line in `AdminSearch.js`? You can pass `${formattedSearchKey}` via another `&lt;input&gt;` tag. And be careful with space and tab characters. We only use 4 spaces here instead of 1 tab character for indentation.